src/js/storage.js

```diff
@@ -395,7 +403,7 @@ BadgerStorage.prototype = {
    * Get an item
    *
    * @param {String} key - the key for the item
-   * @return the value for that key or null
+   * @return {?*} the value for that key or null
```

Comment:
[Clarification] Just as an aside, this return type is unfamiliar to me; can you clarify what this means?

Reply:
I'm mixing the `?` prefix, which signifies something could be null (see the [JSDoc doc. page for @type](http://usejsdoc.org/tags-type.html)) and [this SO answer](https://stackoverflow.com/a/6778666) for how to specify a type that could be anything. This is probably not actually supported ...

Reply:
Haha, fair enough! I guess right now we only return `{?string}`, but absolutely true there's no reason why this couldn't change at some point. Thanks for the explanation :)

Reply:
`BadgerStorage.prototype.getItem` can return any type:
```javascript
//"object"
typeof badger.storage.getBadgerStorageObject('action_map').getItem('outbrain.com')
//"boolean"
typeof badger.getSettings().getItem("seenComic")
```